src/services/task.service.ts

```diff
@@ -0,0 +1,57 @@
+import { DateTime } from 'luxon';
+
+import { taskSchema } from '../models/task.model';
+import { TaskCreatePayload } from '../controllers/v1/task/create.controller';
+import { CreateTaskDto, TaskRepository } from '../repositories/task/create';
+import { TaskStartFinishTimeError, TaskTimelineError } from '../errors/error';
+
+export class TaskService {
+  async createTask(data: TaskCreatePayload) {
+    const taskRepo = new TaskRepository(taskSchema);
```

Comment:
Why not inject this in the constructor so that it is instantiated ones and can easily be tested?